application/app/Http/Requests/DeactivateInstitutionUserRequest.php

```diff
@@ -0,0 +1,75 @@
+<?php
```

Comment:
I feel the same in here as in https://github.com/keeleinstituut/tv-authorization/pull/28#discussion_r1213103163